packages/lit-dev-tools-cjs/src/api-docs/configs/lit-2.ts

```diff
@@ -0,0 +1,194 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as pathlib from 'path';
+
+import type {ApiDocsConfig} from '../types.js';
+
+const root = pathlib.resolve(__dirname, '..', '..', '..', '..', '..');
+const workDir = pathlib.join(root, 'packages', 'lit-dev-api');
+const gitDir = pathlib.join(workDir, 'lit');
+const litDir = pathlib.join(gitDir, 'packages', 'lit');
+const srcDir = pathlib.join(litDir, 'src');
+const outDir = pathlib.join(workDir, 'api-data');
+
+/**
+ * lit.dev API docs configuration for Lit 2.x
+ */
+export const lit2Config: ApiDocsConfig = {
+  repo: 'https://github.com/lit/lit',
+  sha: 'f8ee010bc515e4bb319e98408d38ef3d971cc08b',
```

Comment:
can you add a comment? Is this the SHA the API docs are generated from? If we want to generate docs for a specific release, can this be a sha-like and we use the release tag instead for readability?

Reply:
It's used for the `git checkout` command, and for generating GitHub `blob` URLs, so yeah it can be any commit reference. I renamed the field from `sha` to `commit`, and updated the comment in the interface declaration.